dev_guide/event_notifications.adoc

```diff
@@ -0,0 +1,95 @@
+[[dev-guide-event-notifications]]
+= Event Notifications
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+For each of your projects, you can choose to receive email notifications about
+various xref:../dev_guide/events.adoc#dev-guide-events[events], including:
+
+* _Dead deployment_: A failed deployment in which no other deployment in the
+rollout history is able to remain active. The related deployment configuration
+has no deployments with active replicas.
+
+* _Failed deployment_: A new deployment fails and another deployment in the
+rollout history is now active. The related deployment configuration has one
+active deployment, but it is not the latest deployment.
+
+* _Dead build_: Occurs when automatically triggered builds are failing for a
+specified build configuration that previously had successful builds.
+
+* _Dead persistent volume claim_: A claim that is in the pending state for longer
+than some reasonable amount of time. These are claims whose resource requests
+likely do not match with any existing persistent volumes in the cluster.
+
+* _Lost persistent volume claim_: A claim that is bound to a backing persistent
+volume, but the persistent volume has unexpectedly gone away.
+
+[[event-notifications-how-it-works]]
+== How It Works
+
+The Notifications feature continually monitors events across the cluster. When
+problems are detected, notifications are sent to the email address of the
+project creator, designated in the `requester` field of the project. These
+notifications are throttled so that the recipient is not overwhelmed with email
+messages.
+
+[[event-notifications-configuring-notifications]]
+== Configuring Notifications via the Web Console
+
+From the web console:
+
+. Select *Monitoring* from the left-hand navigation.
+
+. Select *Notifications* from the *Monitoring* dropdown menu.
++
+image::monitoring_menu.png[Monitoring Menu]
+
+. Select your preferences on the *Notifications* page.
++
+image::notifications_UI.png[Notifications Menu]
+
+. Click the *Save* button at the bottom of the page.
++
+Once your options are successfully saved, this message will appear at the top of
+the page:
++
+image::success_message.png[Success Message]
+
+[[event-notifications-setting-preferences]]
+== Setting Preferences
```

Comment:
That sounds appropriate to me. That or maybe just "Preference Storage", since I don't think we want to be encouraging folks to manually modify the ConfigMap.